packages/web3torrent/src/clients/payment-channel-client.ts

```diff
@@ -21,6 +21,11 @@ export interface ChannelState {
   payerBalance: string;
 }
 
+const enum Index {
+  Leecher = 0,
+  Seeder = 1
+}
+
```

Comment:
I like this approach but would prefer to use `payer` and `beneficiary` terminology; this class need not use any torrent specific terms as it could in principle be used in other apps. 

(Also as previously discussed, we can end up with payment channels between peers, none of whom are seeders).

Reply:
See 176c58b